src/VirtualClient/VirtualClient.Actions/Sysbench/SysbenchExecutor.cs

```diff
@@ -48,7 +49,53 @@ public string DatabaseScenario
         {
             get
             {
-                return this.Parameters.GetValue<string>(nameof(this.DatabaseScenario), SysbenchOLTPScenario.Default);
+                return this.Parameters.GetValue<string>(nameof(this.DatabaseScenario), SysbenchScenario.Default);
+            }
+        }
+
+        /// <summary>
+        /// Number of records per table.
+        /// </summary>
+        public int RecordCount
```

Comment:
Code Implementation Recommendation: Avoid significant logic in C# property getters and setters.

Move the logic here for defining the RecordCount to the InitializeAsync() method to be consistent with the relative convention in other VC components.